tests/testsuite/update.rs

```diff
@@ -1661,36 +1689,40 @@ fn update_breaking() {
         .file(
             "Cargo.toml",
             r#"
-            [workspace]
-            members = ["foo", "bar"]
+                # Check if formatting is preserved
 
-            [workspace.dependencies]
-            ws = "1.0"
-        "#,
+                [workspace]
+                members  =  ["foo", "bar"]
+
+                [workspace.dependencies]
+                ws  =  "1.0"  # This line gets partially rewritten
+            "#,
         )
         .file(
             "foo/Cargo.toml",
             r#"
+                # Check if formatting is preserved
+
                 [package]
-                name = "foo"
-                version = "0.0.1"
-                edition = "2015"
-                authors = []
+                name  =  "foo"
+                version  =  "0.0.1"
```

Comment:
Please put all  of these formatting checks in the previous commit

Reply:
Done.